pkg/output/output.go

```diff
@@ -210,6 +216,30 @@ func (b *Handler) writeSIGRetestBadge(name, filePath string, data types.RunningA
 	return badge.Render(name, badgeString, color, f)
 }
 
+func (b *Handler) writeJobFailureBadges(data types.RunningAverageDataItem, levels *Levels) error {
+	failedJobLeaders := data.FailedJobLeaderBoard
+	basePath, err := b.initializeSourcePath()
+	if err != nil {
+		return err
+	}
+	for i, job := range failedJobLeaders {
+		if i < 5 {
```

Comment:
I did not look at the code, but I like the output. I would consider showing the 10 nosiest lanes, not sure.